gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -163,6 +161,89 @@ export default function NxGlobalSidebarPage() {
             </NxTable.Body>
           </NxTable>
         </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxGlobalSidebarFooter</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>showSupport</NxTable.Cell>
```

Comment:
I don't think we should have this property.  Just base it off the `supportLink`

Reply:
> I don't think we should have this property. Just base it off the `supportLink`

Ok, done here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/f6c2dc51636eb3dd205c3ef3bcb6073a9fb19c25

Reply:
> Ok, done here: f6c2dc5

I don't think that's the right commit.  Looking around more broadly though it looks like the prop has been removed but the documentation for it is still there.

Reply:
> > Ok, done here: [f6c2dc5](https://github.com/sonatype/sonatype-react-shared-components/commit/f6c2dc51636eb3dd205c3ef3bcb6073a9fb19c25)
> 
> I don't think that's the right commit. Looking around more broadly though it looks like the prop has been removed but the documentation for it is still there.

Ah right, removed here and I also highlighted that without `supportLink` the `supportText` will not appear. https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/cfe692dc56f8fc65d0451f439156802c00a22d08